support/cas-server-support-pac4j-core-clients/src/main/java/org/apereo/cas/support/pac4j/authentication/DelegatedClientFactory.java

```diff
@@ -390,6 +390,7 @@ protected void configureSamlClient(final Collection<BaseClient> properties) {
                 val cfg = new SAML2Configuration(saml.getKeystorePath(),
                     saml.getKeystorePassword(),
                     saml.getPrivateKeyPassword(), saml.getIdentityProviderMetadataPath());
+                cfg.setCertificateNameToAppend(saml.getClientName());
```

Comment:
It'd be better if you define a standalone separate setting for this, and then default to the client-name if it's undefined.

Reply:
in this case do we need to fallback to the clientName ? The clientName is not a required property and has some "generated parts", like on [at this line](https://github.com/apereo/cas/blob/414f4ecd9084a64309af1b5d762710c89b43ed49/support/cas-server-support-pac4j-core-clients/src/main/java/org/apereo/cas/support/pac4j/authentication/DelegatedClientFactory.java#L442)